lib/home.dart

```diff
@@ -174,23 +182,97 @@ class _HomeState extends State<Home> {
     }
   }
 
+  GlobalKey<NavigatorState> getNavigatorKey(int index) {
+    final auth = Provider.of<AuthModel>(context);
+
+    switch(auth.activeAccount.platform) {
+      case PlatformType.github: {
+        switch(index) {
+          case 0:
+            return tab1;
+          case 1:
+            return tab2;
+          case 2: 
+            return tab3;
+          case 3: 
+            return tab4;
+          case 4: 
+            return tab5;
+        }
```

Comment:
Could we reuse the logic here for other platforms? For example, GitLab would never get into case 3 and 4.

Reply:
Do you mean like writing a single switch statement without checking the platform as for example gitlab would not reach 3,4?

Reply:
Yeah. I'm not sure if there are other side effects in this way.

Reply:
Done.